pallet/deposit/src/lib.rs

```diff
@@ -314,12 +314,12 @@ pub trait SimpleAsset {
 }
 
 /// Deposit identifier.
-///
-/// It's not a global-unique identifier.
-/// It's only used for distinguishing the deposits under a specific account.
+pub type DepositId = u16;
 // https://github.com/polkadot-js/apps/issues/8591
+// Max deposits in Darwinia is 322.
+// Max deposits in Crab is 220.
+// Maybe we will use `WeakBoundedVec` later.
 // pub type DepositId = u8;
-pub type DepositId = u16;
```

Comment:
Why made this change? It leads to data compatible issue both in the Deposit, Staking and precompile side.